app/pb_kits/playbook/plugins/pb_chart_plugin.js

```diff
@@ -0,0 +1,94 @@
+if(typeof(Highcharts) !== 'object') {
+  const script = document.createElement('script')
+  script.src   = 'https://code.highcharts.com/highcharts.js'
+  document.body.appendChild(script)
+}
```

Comment:
This chart plugin is actually 100% reliant on this script. Require it once here only.